main.go

```diff
@@ -26,81 +22,53 @@ func clientSet() *kubernetes.Clientset {
 	return clientSet
 }
 
-func getPods(clientSet *kubernetes.Clientset, options Options) *v1.PodList {
-	excludeValueSet := sets.NewString(options.excludeLabelValue)
-	requirement, err := labels.NewRequirement(options.excludeLabelKey, selection.NotEquals, excludeValueSet)
-	if err != nil {
-		panic(err.Error())
+func getPods(clientSet *kubernetes.Clientset, options options) *v1.PodList {
+	coreClient := clientSet.CoreV1()
+	pods := coreClient.Pods(options.namespace)
+	listOptions := v1.ListOptions{}
+	if options.labelExclusion != nil {
+		selector := labels.NewSelector().Add(*options.labelExclusion)
+		listOptions = v1.ListOptions{LabelSelector: selector.String()}
```

Comment:
Why not change the field of the previous struct instead? `listOptions.LabelSelector = selector.String()`